kani-driver/src/main.rs

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 #![feature(let_chains)]
 #![feature(array_methods)]
-
+#![feature(map_try_insert)]
```

Comment:
Is this needed? Didn't see `try_insert` used in the PR.

Reply:
oh... yeah, that was from a previous iteration.